website/content/docs/concepts/stateful-deployments.mdx

```diff
@@ -0,0 +1,57 @@
+---
+layout: docs
+page_title: Stateful Deployments
+description: |-
+  Learn how Nomad handles Stateful Deployments.
+---
+
+# Stateful Deployments
+
+<Note>
+Stateful deployments support only dynamic host volumes. For CSI volumes, we have
+<a href="/nomad/docs/job-specification/volume#per_alloc">per_alloc</a> property
+that serves a similar purpose.
+</Note>
+
+Nomad stateful deployments allow users to effectively bind their jobs to host
+[volumes]. On a lower level, this means taking Nomad's basic unit of workload
+placement—the [allocation]—and making it possible for users to have allocations
+that are ["sticky"] to volume IDs. Since stateful deployments only support
+dynamic host volumes, using them effectively means binding a job to a particular
+node.
```

Comment:
```suggestion
Nomad stateful deployments let you effectively bind your jobs to host [volumes].
On a lower level, this means taking Nomad's basic unit of workload placement,
the [allocation], and making it possible for you to have allocations that are
[sticky] to volume IDs. Since stateful deployments only support dynamic host
volumes, using them effectively means binding a job to a particular node.
```

style: [address the reader as "you"](https://github.com/hashicorp/tutorials/blob/main/STYLE_GUIDE.md#use-the-grammatical-person-you-for-the-practitioner)